src/lib.rs

```diff
@@ -708,18 +613,114 @@ impl vfs::filesystem::FileSystem for FRFS {
 }
 
 /// Load a FRFS file.
-pub fn load<P: AsRef<path::Path> + Copy>(path: P) -> Result<FRFS> {
+pub fn load(path: impl AsRef<Path>) -> Result<FRFS> {
     FRFS::new(path)
 }
 
 /// Load a FRFS file from frfs::File.
-pub fn load_from_embed_file(file: File) -> Result<FRFS> {
-    FRFS::from_embed_file(file)
+pub fn load_from_file(file: File) -> Result<FRFS> {
+    FRFS::from_file(file)
+}
+
+#[derive(Debug)]
+pub struct FRFSBuilder {
+    header: Header,
+    source: PathBuf,
+}
+
+impl FRFSBuilder {
```

Comment:
build is missing in public method

Reply:
Also, is there any benefit to standalone FRFSBuilder?

Reply:
To simply answer the benefit: we could separate the builder and the readonly filesystem. They should not be mixed. The mixture of changing on the disk and building to a frfs file will cause many corner cases.